docs/manual-EN/5.appendix/vid-partition.md

```diff
@@ -0,0 +1,25 @@
+# Vertex Identifier and Partition
+
+This document provides some introductions on vertex identifier (`VID` for short) and partition.
+
+In **Nebula Graph**, vertices are identified with vertex identifiers (i.e. `VID`s). When inserting a vertex, you must specify a `VID` for it. You can generate `VID`s either with your own application or with the hash function provided by **Nebula Graph**.
+
+`VID`s must be unique in a graph space. That is, in the same graph space, vertices with the same `VID` are considered as the same vertex. `VID`s in different graph spaces are independent of each other. In addition, one `VID` can have multiple `TAG`s.
+
+When inserting data into **Nebula Graph**, vertices and edges will distribute to different partitions. And the partitions are located on different machines. If you want some certain vertices to locate on the same partition (i.e. on the same machine), you can control the generation of the `VID`s by using the following formula.
```

Comment:
When inserting data into **Nebula Graph**, vertices and edges will distribute to different partitions.  ==> When inserted into **Nebula Graph**, vertices and edges are distributed across different partitions.